common.py

```diff
@@ -0,0 +1,52 @@
+import os
+from brownie import *
+import brownie
+from ape_safe import ApeSafe
```

Comment:
We can remove this, right, since we aren't using ape safe?

Reply:
yup thanks, removed